docs/FAQ.md

```diff
@@ -17,6 +17,33 @@ with `collect`, `show` or `write` a new `DataFrame` is constructed causing Spark
 query. This is why `spark.rapids.sql.enabled` is still respected when running, even if explain shows
 stale results.
 
+### Why does the plan for the GPU query look different from the CPU query?
+
+Typically, there is a one to one mapping between CPU stages in a plan and GPU stages.  There are a
+few places where this is not the case.
+
+* `WholeStageCodeGen` - The GPU plan typically does not do code generation, and does not support generating code for an
```

Comment:
I'd like to keep the line lengths to 100 characters for ease of editing, where possible.

Reply:
I can see us trying to limit ourselves to 100 characters most of the time, I have real problems from a change management perspective when we try to make all of the lines as close to 100 characters as possible, like most of this file is.  When you make a single small change to a line that forces it over 100 characters instantly you feel like you need to reformat the rest of the paragraph to make all of the lines as close to 100 characters as possible.  This means git sees a much bigger change and trying to merge conflicting changes becomes a real pain.  Especially because all of those changes have no impact to what the final rendered result looks like.